python/torch_mlir_e2e_test/test_suite/basic.py

```diff
@@ -304,6 +304,29 @@ def AddmmModule_differentRankBroadcastable(module, tu: TestUtils):
 # ==============================================================================
 
 
+class UnflattenStaticModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+        self.unflat = torch.nn.Unflatten(1, (2, 3))
+
+    @export
+    @annotate_args([
+        None,
+        ([1, 6, 4], torch.float32, True),
+    ])
+    def forward(self, x):
+        return self.unflat(x)
```

Comment:
nit: can you use `torch.ops.aten.unflatten(x, 1, (2, 3))` to make it clear that `aten.unflatten` is being tested? See: https://github.com/llvm/torch-mlir/blob/main/docs/adding_an_e2e_test.md#forward-method